helpers.go

```diff
@@ -143,6 +144,28 @@ func defaultString(value string, defaultValue []string) string {
 	return value
 }
 
+// defaultInt returns the value or a default value if it is set.
+func defaultInt(value string, defaultValue []int) int {
```

Comment:
defaultInt should have an int as first value and the conversion should take place outside of it